pkg/kubernetes/cluster.html

```diff
@@ -224,6 +224,63 @@ <h4 class="modal-title" translatable="yes">Adjust Service</h4>
       </div>
     </div>
 
+    <div class="modal" id="adjust-entity-dialog" type="text/ng-template" tabindex="-1" role="dialog" data-backdrop="static">
```

Comment:
Why is this type="text/ng-template"? This doesn't seem to be an angular template. Also, as far as I know, that's not the correct syntax for such a template.